docs/sphinx/developers/building-bioformats.txt

```diff
@@ -114,11 +114,40 @@ and JAR dependencies (stored in the Maven repository in
 :file:`~/.m2/repository`) on the build path, depending on which projects are
 currently open.
 
-We recommend using Eclipse 4.3 (Kepler), specifically -
+We recommend using Eclipse 4.3 (Kepler) or later, specifically -
 "Eclipse IDE for Java developers". It comes with m2e installed
 (http://eclipse.org/downloads/compare.php?release=kepler).
 
-You can then import the Bio-Formats source by choosing
+Bio-Formats can be included as a dependency in any Maven project. The 
+dependency can be added to the project pom file and should include the
+desired Bio-Formats version. Using bioformats_package as the artifactId
+will include the complete Bio-Formats package, or individual components 
+can be chosen as desired.
+
+::
+
+  <dependency>
+    <groupId>OME</groupId>
+    <artifactId>bioformats_package</artifactId>
+    <version>5.2.0</version>
+  </dependency>
+
+In order to include this Bio-Formats dependency a custom repository 
+must also be added to the project pom or ${user.home}/.m2/settings.xml 
+The repositories element is inherited so for a group of projects the 
+repositories element can be defined at the top of your inheritance chain. 
+
+::
+
+  <repositories>
+    <repository>
+      <id>OME</id>
+      <name>Bio-Formats Repo</name>
+      <url>http://artifacts.openmicroscopy.org/artifactory/repo</url>
```

Comment:
Having looked a bit further at the Artifactory documentation, `/repo` is the URL suffix used for the [Virtual repository](https://www.jfrog.com/confluence/display/RTF/Virtual+Repositories) which aggregates *all* local and remote repositories. As this might include unwanted/experimental JARs (in our case `ome.unstable/ome.experimental`), I think we should rather expose to the `/maven` URL.